### ISSUE TITLE
Documentation Error

### ISSUE THREAD

**Issue by luizfelipevbll** [encode/django-rest-framework#7932]
I keep receiving this errors when following the [DRF docs](https://www.django-rest-framework.org/topics/documenting-your-api/#a-minimal-example-with-swagger-ui) to generate the documentation.
Error on redoc:
![image](https://user-images.githubusercontent.com/40257388/115240209-e4251800-a0f5-11eb-946c-a145c108606b.png)
Error on sawgger:
![image](https://user-images.githubusercontent.com/40257388/115240243-ebe4bc80-a0f5-11eb-8695-20695a095022.png)

Here is my code:

generator = schemas.SchemaGenerator(title='Documentação', version='1.0.0')
```python3
@api_view(http_method_names=['GET'])
@renderer_classes([renderers.OpenAPIRenderer])
@authentication_classes([BasicAuthentication, SessionAuthentication])
@permission_classes([IsAuthenticated, PermReadDocs])
def open_api(request):
    schema = generator.get_schema(request)
    return response.Response(schema)


@api_view(http_method_names=['GET'])
@authentication_classes([BasicAuthentication, SessionAuthentication])
@permission_classes([IsAuthenticated, PermReadDocs])
def read_docs(request):
    return SimpleTemplateResponse('swagger-ui.html', {'schema_url': 'app_documentacao:openapi-schema'})
```
The docs.yaml are beeing generated, but It seens they can not be interpreted.
![image](https://user-images.githubusercontent.com/40257388/115240552-39f9c000-a0f6-11eb-96f4-dd497eb9d85c.png)

**Comment by luizfelipevbll** [comment]
**unknown tag !<tag:yaml.org,2002:python/tuple>**

**Comment by JulienPalard** [comment]
`SchemaGenerator` is documented to be:

> A class that walks a list of routed URL patterns, requests the schema for each view, and collates the resulting CoreAPI Document.

However CoreAPI is [no longer maintained](https://github.com/core-api/core-api):
> DEPRECATION NOTICE: As of djangorestframework 3.9, the plan is to phase out CoreAPI in favor of OpenAPI as the default schema representation.

The page mentionning SchemaGenerators is no longer listed in the docs (no links pointing to it in the menu), but it still exists.

Should the [schema](https://www.django-rest-framework.org/coreapi/schemas/) page explicitly mention it's deprecated? Along with other pages in `/coreapi/`?

**Comment by JulienPalard** [comment]
To generate an OpenAPI Schema use: https://www.django-rest-framework.org/api-guide/schemas/.

**Comment by luizfelipevbll** [comment]
> `SchemaGenerator` is documented to be:
> 
> > A class that walks a list of routed URL patterns, requests the schema for each view, and collates the resulting CoreAPI Document.
> 
> However CoreAPI is [no longer maintained](https://github.com/core-api/core-api):
> 
> > DEPRECATION NOTICE: As of djangorestframework 3.9, the plan is to phase out CoreAPI in favor of OpenAPI as the default schema representation.
> 
> The page mentionning SchemaGenerators is no longer listed in the docs (no links pointing to it in the menu), but it still exists.
> 
> Should the [schema](https://www.django-rest-framework.org/coreapi/schemas/) page explicitly mention it's deprecated? Along with other pages in `/coreapi/`? Should https://www.django-rest-framework.org/topics/api-clients/ be removed too?

Thank you for the help! 
Would be great if those deprecated were removed or receive a explicit flag that are no longer supported.

**Comment by cwkang1998** [comment]
I can help contribute to this documentation change if its not being worked on / taken yet.

**Comment by tomchristie** [comment]
Closed via this docs change: #8186
